starter-code/css/main.css

```diff
@@ -0,0 +1,119 @@
+* {
+    font-family: "Lato", sans-serif;
+}
+
+body {
+    max-width: 80%;
+    margin: 10% auto;
+}
+
+img {
+    width: 100%;
+}
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-weight: 100;
+    margin-bottom: -8px;
```

Comment:
This is definitely a valid property value, but usually you want to avoid negatives values as it could lead to other issues.